Source/DafnyLanguageServer/Language/SyntaxTreeVisitor.cs

```diff
@@ -445,9 +452,21 @@ public abstract class SyntaxTreeVisitor {
         case ITEExpr ifThenElseExpression:
           Visit(ifThenElseExpression);
           break;
+        case LambdaExpr lambdaExpr:
```

Comment:
I see that it appears to be a bit repetitive. However, the current design creates a `Visit` overload for each type. It could be confusing now that you handle each type explicitly but call a common overload for the base type `ComprehensionExpr`.

Reply:
Agree. I factored these five cases into a single case.